tools/osbuild-depsolve-dnf

```diff
@@ -272,7 +275,7 @@ class Solver():
             pkg_repos[pkgrepo.id] = pkgrepo
 
         repositories = {}  # full repository configs for the response
-        for _, repo in pkg_repos:
+        for _, repo in pkg_repos.items():
```

Comment:
I guess you can iterate over `.values()`?

Reply:
wow, big Golang brain there.

Reply:
Also that's on the wrong commit!

Reply:
Fixed.